ua-snap/usda-dash
# Update text and logo

Closes #5 , #4 .

Also updates the pipfile to correctly reflect contents of requirements.txt.  (Was getting a broken build when just doing the standard install, package dependencies were not specified correctly).

To test this:

```
pipenv --rm
git checkout update_text_and_logo
pipenv install
pipenv run python index.py
```

The environment should function without error.

Check for added AK CASC logo and modified text in the footer.
